resources/modes/print/print.js

```diff
@@ -30,10 +30,21 @@ $(function() {
           .text(robopaint.t('common.action.start'));
         $('#buttons button.normal').prop('disabled', false); // Enable options
         $('#cancel').prop('disabled', true); // Disable the cancel print button
+        $("#processing-dialog").hide();
```

Comment:
The only situation where the processing might continue until this point is if didn't actually generate any commands for the entire run. I can't say I call that a good thing.. but I suppose it's better than having the dialog simply stuck on the users screen in the event that no events are ever sent to the bot. Both equally unlikely events :ocean: